salt/modules/cmdmod.py

```diff
@@ -3663,6 +3663,11 @@ def powershell(
 
         salt '*' cmd.powershell "$PSVersionTable.CLRVersion"
     """
+    if shell not in ["powershell", "pwsh"]:
+        msg = "Must specify a valid powershell binary. Must be 'powershell' " \
+              "or 'pwsh'"
+        raise CommandExecutionError(msg)
```

Comment:
Why not `raise CommandExecutionError("The Message")`?
Why create a `msg` variable for a single use?

Reply:
Done